todo-vue/src/components/Home.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="home">
+    <header><h1>Todo App</h1></header>
+    <main>
+      <ul>
+        <CardComponent
+          v-for="todo in todos"
+          :key="todo.id"
+          :title="todo.title"
+          :id="todo.id"
+          @setTitle="editTodo"
+          @delete="deleteTodo"
```

Comment:
```suggestion
          @edit="editTodo"
          @delete="deleteTodo"
```
vue에서는 보통 이렇게 `snake-case`로 표기하는게 컨벤션이랍니다!